lib/ethdev/rte_ethdev.h

```diff
@@ -174,6 +174,7 @@ extern "C" {
 
 #include "rte_ethdev_trace_fp.h"
 #include "rte_dev_info.h"
+#include "rte_ethdev_netns.h"
```

Comment:
Please remove this. I think you've forgotten it here.

Reply:
I suspect it's not forgotten.

Reply:
No, it's not. the include is needed for `RTE_ETH_NETNS_MAX_LEN`